lib/kadai-core/src/main/java/io/kadai/workbasket/internal/WorkbasketAccessSqlProvider.java

```diff
@@ -175,12 +175,12 @@ private static String insertValues() {
         .collect(Collectors.joining(", ", "", " "));
   }
 
-  private static String getMaximumPermissionStatement(boolean isNotDb2AndNotOracle) {
+  private static String getMaximumPermissionStatement(boolean isNotDb2) {
```

Comment:
Why do we pass here a boolean instead of the enum "DB"?

Reply:
Because we don't know which DB is used when calling the method. It's only known that it's not DB2

Reply:
It is not a blocking issue, but I am just curious, there is absolutely no way to find it, as a parameter to the method or in a @PostConstruct method in the class?